### PR TITLE
add doc for mod_tcp_keepalive

### PR DESCRIPTION
**Pull request by dblate**
Signed-off-by: yuhui06 <yuhui06@baidu.com>

### CODE REVIEW

**Comment by dblate** [comment]
include en_us doc

File: docs/zh_cn/modules/mod_tcp_keepalive/mod_tcp_keepalive.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+## 规则配置
+
+### 配置描述
+
+| 配置项 | 描述 |
+| ----- | --- |
+| Version | String<br>配置文件版本 |
+| Config | Object<br>各产品线（租户）的TCP心跳包管理规则 |
+| Config{k} | String<br>产品线名称 |
+| Config{v} | Array<br>产品线下规则列表 |
```
Comment (mileszhang2016) [inline docs/zh_cn/modules/mod_tcp_keepalive/mod_tcp_keepalive.md:38]: 产品线下规则列表 => 产品线的规则列表

File: docs/en_us/modules/mod_tcp_keepalive/mod_tcp_keepalive.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+### Description
+
+| Config Item | Description |
+| ----- | --- |
+| Version | String<br>Version of config file |
+| Config | Object<br>Rules for each product |
+| Config{k} | String<br>Product name |
+| Config{v} | Array<br>A list of rules |
+| Config{v}[] | Object<br>A specific rule |
+| Config{v}[].VipConf | Array<br>The IP list to set the keepalive message strategy  |
```
Comment (mileszhang2016) [inline docs/en_us/modules/mod_tcp_keepalive/mod_tcp_keepalive.md:40]: => 
The list of virtual IPs to set the keepalive message strategy

File: docs/zh_cn/modules/mod_tcp_keepalive/mod_tcp_keepalive.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+| Config{v}[] | Object<br>某一条规则详细信息 |
+| Config{v}[].VipConf | Array<br>需要配置的IP数组，数组中的IP共用以下同一套策略 |
```
Comment (mileszhang2016) [inline docs/zh_cn/modules/mod_tcp_keepalive/mod_tcp_keepalive.md:40]: 需要配置的VIP（Virtual IP）数组，数组中的VIP共用以下同一套策略

**Review by mileszhang2016 — CHANGES_REQUESTED** [review]

**Pull request merged**